src/app/components/elements/panels/EventOverviews.tsx

```diff
@@ -27,84 +27,240 @@ export const EventOverviews = ({setSelectedEventId, user}: {user: PotentialUser;
         dispatch(getEventOverviews(overviewFilter));
     }, [dispatch, setSelectedEventId, overviewFilter]);
 
-    return <Accordion trustedTitle="Events overview" index={0}>
-        {isEventLeader(user) && <div className="bg-grey p-2 mb-4 text-center">
-            As an event leader, you are only able to see the details of events which you manage.
-        </div>}
+    function eventAttendancePercentage(
+      numberAttended: number,
+      numberAbsent: number
+    ) {
+      const percentage =
+        (numberAttended / (numberAttended + numberAbsent)) * 100;
+      if (isNaN(percentage)) return "-";
+      else return percentage.toFixed(1) + "%";
+    }
+
+    return (
```

Comment:
I think there may be something wrong with your Prettier configuration, because part of this file has been refactored to adhere to some code style, but it is mixing 2-space and 4-space indentation. I suggest implementing the Prettier ticket as a priority and in the meanwhile deactivate Prettier in your IDE, because these style changes are not correct.